src/translations/en_US/content/astrolich/victrixLudorum/messages.json

```diff
@@ -0,0 +1,18 @@
+{
+    "content.astrolich.victrixLudorum": "Victrix Ludorum",
+
+    "content.astrolich.victrixLudorum.hide.title": "Hide",
+    "content.astrolich.victrixLudorum.hide.description": "You have learned the best way to attack your enemies is when they are looking the other way. You coneal yourself at a DR10.",
```

Comment:
Grammar: "the best way" doesn't match "is when". Maybe it should be "the best time" ?